src/hubmap_translation/addl_index_transformations/portal/utils.py

```diff
@@ -6,5 +6,6 @@
 
 
 def _log_transformation_error(doc, msg):
+    doc['transformation_errors'] = doc.get('transformation_errors', [])
```

Comment:
When did you run into an issue that prompted this? `transformation_errors` is set in https://github.com/hubmapconsortium/search-api/blob/2c0673cd7ff33cf525815f807ca124f132a9daff/src/hubmap_translation/addl_index_transformations/portal/__init__.py#L72.